src/main/java/nextstep/qna/domain/Answers.java

```diff
@@ -0,0 +1,26 @@
+package nextstep.qna.domain;
+
+import nextstep.users.domain.NsUser;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Answers {
+    private List<Answer> answers = new ArrayList<>();
```

Comment:
```suggestion
public class Answers {
    private final List<Answer> answers = new ArrayList<>();
```
일급컬랙션 👍 

불변하게 유지하면 좋을 것 같아요 😄